### PR TITLE
New django module

### PR DESCRIPTION
**Pull request by stefdworschak**
-) Updating enrollment script as to not enroll students in the old Django module
-) Adding script to deactivate one course enrollment and replace it by enrolling them in another for a list of students

Before deploying this to production the steps captured in this document will have to be followed:
https://docs.google.com/document/d/1ollnp0iIwgdXjMBFaX3_HgSw2E5JYB7ZW3uBFXMJEQQ/edit#

### CODE REVIEW

File: common/djangoapps/enrollment/management/commands/replace_enrollments.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+from enrollment.api import add_enrollment
+from enrollment.errors import CourseEnrollmentExistsError
+from opaque_keys import InvalidKeyError
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+
+def replace_course_enrollment(student, deactivate_enrollment,
+                              replace_with_enrolment=None, mode=None):
```
Comment (nakitamccool) [inline common/djangoapps/enrollment/management/commands/replace_enrollments.py]: Minor - Please update to replace_with_enrollment, so we've standardised spelling across functions.

File: common/djangoapps/enrollment/management/commands/replace_enrollments.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            Example course id: 'course-v1:CodeInstitute+F101+2017_T1'
+
+        """
+        successful_changes = 0
+        filepath = options.get('filepath')
+        try:
+            if filepath is None:
+                raise IOError('No filepath specified. Please add the -f or --filepath flag.')
+
+            df = pd.read_csv('replace_file.csv')
```
Comment (nakitamccool) [inline common/djangoapps/enrollment/management/commands/replace_enrollments.py]: Could we update the filename to something more descriptive - e.g. substitute_enrollments.csv ?

**Review by nakitamccool — APPROVED** [review]
LGTM - few minor comments on naming but all good otherwise :)

File: common/djangoapps/enrollment/management/commands/replace_enrollments.py (hunk 1 above)
Comment (stefdworschak) [reply common/djangoapps/enrollment/management/commands/replace_enrollments.py]: Thanks for highlighting this, I did not see the spelling mistake. Fixed the spelling mistake everywhere.

File: common/djangoapps/enrollment/management/commands/replace_enrollments.py (hunk 2 above)
Comment (stefdworschak) [reply common/djangoapps/enrollment/management/commands/replace_enrollments.py]: Oh, this should not actually be hardcoded, but come from the flag in the cli argument. Fixed this to work as expected.

**Pull request merged**